app/components/dashboard/in_progress_collection_row_component.html.erb

```diff
@@ -1,9 +1,11 @@
 <div class="row">
   <div class="col-4">
-    <%= show_collection_link %>
+    <%= show_collection_link %> <%= helpers.turbo_frame_tag dom_id(collection, :edit),
+    src: edit_link_collection_path(collection),
+    target: '_top' %>
   </div>
   <div class="col-3">
-    <%= link_to 'Complete draft', edit_collection_version_path(collection.head), class: "mb-2 btn btn-primary" %>
+    <%= link_to 'Continue editing or deposit', edit_collection_version_path(collection.head), class: "mb-2 btn btn-primary" %>
```

Comment:
should the class be `btn-outline-primary` instead of `btn-primary` to match the mockup on the ticket?

Reply:
@amyehodge  I believe the text should be "Edit or deposit" for consistency. (See https://github.com/sul-dlss/happy-heron/issues/1390#issuecomment-840634899)

Reply:
@jcoyne updated.